src/tactic/ui/tools/ingest_wdg.py

```diff
@@ -1824,6 +1837,9 @@ def execute(my):
                 process = "publish"
 
 
+            if update_sobject_found and update_process == "update":
+                process = "update"
+
```

Comment:
that looks wrong
why is it checking for a particular value "update", isn't this meant to just do 
process = update_process